packages/flutter_hooks/lib/src/listenable.dart

```diff
@@ -128,3 +128,68 @@ class _UseValueNotifierHookState<T>
   @override
   String get debugLabel => 'useValueNotifier';
 }
+
+/// Adds a given [listener] to a [Listenable] and removes it when the hook is
+/// disposed.
+///
+/// As opposed to `useListenable`, this hook does not mark the widget as needing
+/// build when the listener is called. Use this for side effects that do not
+/// require a rebuild.
+///
+/// See also:
+///  * [Listenable]
+///  * [ValueListenable]
+///  * [useListenable]
+void useOnListenableChange(
+  Listenable? listenable,
+  VoidCallback listener, {
+  List<Object?>? keys,
```

Comment:
Keys aren't necessary.

And we should naturally support `listener` changing.

Reply:
Good catch, oversight on my end